tf-generator/aws-services/redis.go

```diff
@@ -186,9 +187,16 @@ func generateRedisVars(duplo duplosdk.DuploEcacheInstance, prefix string) []comm
 	}
 	varConfigs["size"] = var2
 
+	ver := duplo.EngineVersion
+	if duplo.CacheType == 0 {
+		verSplit := strings.Split(duplo.EngineVersion, ".")
+		if verSplit[1] == "0" || verSplit[2] == "0" {
```

Comment:
verSplit[1] == "0" || verSplit[2] == "0" is this correct or we need length >2 && verSplit[2] == "0"